apps/agreement/contracts/Agreement.sol

```diff
@@ -409,6 +422,7 @@ contract Agreement is IArbitrable, ILockManager, IAgreement, IACLOracle, AragonA
     * @param _actionId Identification number of the action to be settled
     */
     function settleAction(uint256 _actionId) external {
+        // Note: (for all users of getChallengedAction()) the errors emitted will be different based on whether the action had a previous challenge
```

Comment:
Ah no, I mean that there are two different errors that can get thrown here:

1. `AGR_CHALLENGE_DOES_NOT_EXIST`
2. `AGR_CANNOT_SETTLE_ACTION`

Reply:
I think that's okay and actually provides a more descriptive context about the current status of the action, do you see any issues with it?

Reply:
Hmm interesting point. I was thinking that it would be easier to describe behaviour if we just had a single error for this, but it's true that this actually gives you more useful information than just a single error. Let's leave it like this 👍.